src/picongpu/include/simulationControl/MySimulation.hpp

```diff
@@ -480,6 +483,28 @@ class MySimulation : public SimulationHelper<simDim>
     {
         namespace nvfct = PMacc::nvidia::functors;
 
+        typedef typename PMacc::particles::traits::FilterByIdentifier
+        <
+            VectorAllSpecies,
+            momentumPrev1
+        >::type VectorSpeciesWithMementumPrev1;
+
+        /* copy attribute momentum to momentumPrev1 */
+        ForEach<
+            VectorSpeciesWithMementumPrev1,
+            particles::Manipulate<
+                particles::manipulators::CopyAttribute<
+                    momentumPrev1,
+                    momentum
+                >,
+                bmpl::_1
+            >
+        > copyMomentumPrev1;
+        copyMomentumPrev1(
+            forward( particleStorage ),
```

Comment:
@psychocoderHPC careful, this class is removed in `dev` now. access species via `DataConnector` :)